cfgov/jinja2/v1/_includes/organisms/form-explainer.html

```diff
@@ -0,0 +1,198 @@
+    {{ render_term( item, page_num, category_index, loop.index ) }}
+{% endfor %}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_term( note, page_num, category_index, note_index )
+
+   ==========================================================================
+
+   Description:
+
+   Render an o-expandable note with a definition.
+
+   item:      A note object. Note objects have the following structure:
+              {
+                'heading': 'Note title.',
+                'body':    'Text of note.'
+              }
+
+   page_num:   Page containing note.
+
+   category_index: Category containing note.
+
+   note_index: index of note in category note array.
+
+
+   ========================================================================== #}
+
+{% macro render_term( item, page_num, category_index, note_index ) %}
+{% set id = 'page-' + page_num | string + '-category-' + category_index | string + '-note-'  + note_index | string %}
+<div class="o-expandable
+            o-expandable__padded
+            o-expandable__form-explainer
+            o-expandable__form-explainer-category-{{ category_index }}"
+     id="{{ id }}">
+    <button class="o-expandable_header o-expandable_target"
+            tabindex="0">
+        <span class="h4 o-expandable_header-left o-expandable_label">
+            {{ item.heading }}
+        </span>
+        <span class="o-expandable_header-right o-expandable_link">
+            <span class="o-expandable_cue-open" aria-label="Show">
+                {{ svg_icon('plus-round') }}
+            </span>
+            <span class="o-expandable_cue-close" aria-label="Hide">
+                {{ svg_icon('minus-round') }}
+            </span>
+        </span>
+    </button>
+    <div class="o-expandable_content" tabindex="0">
+        {{ item.body | safe }}
+    </div>
+</div>
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_pagination( pages )
+
+   ==========================================================================
+
+   Description:
+
+   Render pagination.
+
+   pages: Pages in form. The data structure is as follows:
+          [
+            { 'image':   wagtail image,
+              'categories': [ ... ]
+            }
+          ]
+
+   ========================================================================== #}
+
+{% macro render_pagination( pages ) %}
+<nav class="pagination explain_pagination">
+    <span class="form-explainer_nav-label">Viewing page:</span>
+    <ul class="form-explainer_page-links">
+        {% for page in pages %}
+        <li>
+            <a class="form-explainer_page-link {{ 'current-page' if loop.index == 1 else '' }}"
+               data-page="{{ loop.index }}">
+               {{ loop.index }}
+            </a>
+        </li>
+        {% endfor %}
+    </ul>
+</nav>
+{% endmacro %}
+
+
+
+<div id="form-explainer" class="form-explainer">
```

Comment:
Does this `id` need to be more unique? Can you add more than one form explainer per page?

Reply:
That's a good point. Technically you can add multiple explainers, although that's not something we'd generally (maybe ever?) want to do. Even if that's our guidance, though, I'll need to either limit to one explainer or make these ids unique and update the explainer init code to handle multiples.

Reply:
I added validation that limits explainer blocks to one per page in https://github.com/cfpb/cfgov-refresh/pull/5491/commits/ab6e249f934e6676e9d34bc422f71d6b5728b0ea. (I still need to write tests)